src/awesome_map/BoundaryInterceptor.js

```diff
@@ -232,10 +322,18 @@ define(function(require) {
                 this._touchState = targetState;
                 break;
 
-            case EventTypes.DRAG:
             case EventTypes.DRAG_END:
+                /* jshint -W086 */// Expected break statement 
+
+                this._debouncedDetermineBoundaryVisibility();
```

Comment:
This feels a little off to me.  With a swipe or a drag, there is usually only one drag end event, after a whole bunch of drag events.  Why would you debounce?